extensions/applicationinsights-cfgsync-js/src/CfgSyncPlugin.ts

```diff
@@ -121,6 +122,7 @@ export class CfgSyncPlugin extends BaseTelemetryPlugin implements ICfgSyncPlugin
                 _self._addHook(onConfigChange(config, () => {
                     let ctx = createProcessTelemetryContext(null, config, core);
                     _extensionConfig = ctx.getExtCfg(identifier, _defaultConfig);
+                    _blockCdn = !!_extensionConfig.blockCdn;
```

Comment:
I know that I like to keep the names short, but we probably should identify that this is for fetching the "config" from the cdn and not anything else (like loading the SDK from the Cdn). So maybe `blkCdnCfg` or `disableCdnCfg`